plugins/catalog/src/components/CatalogTable/CatalogTable.tsx

```diff
@@ -72,6 +72,7 @@ export const CatalogTable = (props: CatalogTableProps) => {
       ...createEntitySpecificColumns(),
       columnFactories.createMetadataDescriptionColumn(),
       columnFactories.createTagsColumn(),
+      columnFactories.createTitleColumn(),
```

Comment:
I'd say from an end user perspective, it's probably pretty unexpected to for example go in their code editor and getting nice tab completion for `columnFactories.createTitleColumn()` but then struggling with it just not showing up for unknown reasons. :)

Would you mind adding an `options?: { hidden?: boolean }` to this function and making sure to set that to true in the other files above? That would leave the "expected" use case intact.

Reply:
Makes sense, changed the order of the columns too, as it makes more sense to have title before name.